core/src/SYCL/Kokkos_SYCL_Parallel_Team.hpp

```diff
@@ -403,14 +403,14 @@ class ParallelFor<FunctorType, Kokkos::TeamPolicy<Properties...>,
 
       cgh.parallel_for(
           sycl::nd_range<2>(
-              sycl::range<2>(m_league_size * m_team_size, m_vector_size),
-              sycl::range<2>(m_team_size, m_vector_size)),
+              sycl::range<2>(m_vector_size, m_league_size * m_team_size),
+              sycl::range<2>(m_vector_size, m_team_size)),
```

Comment:
that doesn't make sense, the vector needs to be on the right most side or? Vector threads need to be closest.

So I was thinking:
```
sycl::range<2>(m_team_size, m_league_size * m_vector_size),
sycl::range<2>(m_team_size, m_vector_size))
```